capabilities/display_capabilities.js

```diff
@@ -1264,12 +1109,7 @@ SDL.templateCapabilities = {
                 }
             }
         },
-        "buttonCapabilities": [{
-            "name": "CUSTOM_BUTTON",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }],
+        "buttonCapabilities": SDL.ButtonCapability.filter(button => button.name === 'CUSTOM_BUTTON'),
```

Comment:
```suggestion
        "buttonCapabilities": [...SDL.NAVButtonCapability, ...SDL.ButtonCapability.filter(button => button.name === 'CUSTOM_BUTTON')],
```
Seems that the NAV button capabilities should be included here. Otherwise where are they supported?

Reply:
@jacobkeeler And [here on the develop branch](https://github.com/smartdevicelink/sdl_hmi/blob/d58988ec3e1d00f619bcb88d68f5e3aff2fec312/capabilities/display_capabilities.js#L1267) is only `CUSTOM_BUTTON`.
Are you sure it should be extended?

Reply:
The capabilities on the develop branch definitely seem to be incorrect in this case. All of the NAV buttons are included in the main button capabilities, but aren't included in any individual template's capabilities. This template allows an app to subscribe to these buttons, so they should be included in the capabilities.

Reply:
@jacobkeeler Implemented 5563ddf80f22964bb8696a1735d8f2604accec06